ZomBerry/Addons/scripts/5_Mission/ZomBerryMenu.c

```diff
@@ -497,6 +530,12 @@ class ZomberryMenu extends UIScriptedMenu {
 			}
 			m_lastSelObj = "";
 		}
+		
+		if ( w == m_MapWidget ) {
+			ZomberryBase.Log("", "scale: " + m_MapWidget.GetScale());
```

Comment:
This can be removed

Reply:
Aight, lets remove it before I merge this then :)